migrations/20221125140132_backfill_email_identity.up.sql

```diff
@@ -1,11 +0,0 @@
--- backfill the auth.identities column by adding an email identity 
--- for all auth.users with an email and password 
-
-do $$
-begin
-	insert into {{ index .Options "Namespace" }}.identities (id, user_id, identity_data, provider, last_sign_in_at, created_at, updated_at)
-	select id, id as user_id, jsonb_build_object('sub', id, 'email', email) as identity_data, 'email' as provider, null as last_sign_in_at, '2022-11-25' as created_at, '2022-11-25' as updated_at
-	from {{ index .Options "Namespace" }}.users as users
-	where encrypted_password != '' and email is not null and not exists(select user_id from {{ index .Options "Namespace" }}.identities where user_id = users.id);
-end;
-$$;
```

Comment:
Shall we keep the file but delete / comment out the sql statement? Not sure if deleting the file would break the migrations table of projects that already applied all migrations.

Reply:
i'm not sure if we should be doing anything here - the root cause is due to external factors truncating the `auth.schema_migrations` table and this just seems like a hacky fix to hide the problem